FirmwareGen/CommandLine/GenerateWindowsFFUOptions.cs

```diff
@@ -5,31 +5,31 @@ namespace FirmwareGen.CommandLine
     [Verb("generate-windows-ffu", HelpText = "Generates a ffu from a Windows Base VHDX")]
     public class GenerateWindowsFFUOptions
     {
-        [Option('d', "driver-pack", HelpText = "Todo", Required = true)]
+        [Option('d', "driver-pack", HelpText = "Driver pack with device.xml", Required = true)]
         public string DriverPack
         {
             get; set;
         }
 
-        [Option('o', "output", HelpText = "Todo", Required = true)]
+        [Option('o', "output", HelpText = "Output folder", Required = true)]
         public string Output
         {
             get; set;
         }
 
-        [Option('v', "windows-ver", HelpText = "Todo", Required = true)]
+        [Option('v', "windows-ver", HelpText = "Version of Windows e.g. 10.0.26063.1", Required = true)]
         public string WindowsVer
         {
             get; set;
         }
 
-        [Option('w', "windows-dvd", HelpText = "Todo", Required = true)]
+        [Option('w', "windows-dvd", HelpText = "Windows Setup Media DVD drive", Required = true)]
         public string WindowsDVD
         {
             get; set;
         }
 
-        [Option('i', "windows-index", HelpText = "Todo", Required = true)]
+        [Option('i', "windows-index", HelpText = "Index of windows, professional or other, e.g. 1", Required = true)]
```

Comment:
Small nit but I would word this differently because this is about the index in the install wim image, which isnt really directly corelated with different editions, and may even be the same edition for that matter.

I would go more with something like this instead:

The index of the Windows Setup Media DVD Install Image, to apply. e.g. 1